bindata/network/openshift-sdn/cni-features.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ConfigMap
+apiVersion: v1
+immutable: true
+metadata:
+  name: openshift-network-features
+  namespace: openshift-config-managed
+  annotations:
+    openshift.io/description: |
+      Exposes available network features as required by the Console in order to show or hide some form fields.
+      If the map or a given property is undefined, the Console won't throw error and will take a default action (show, hide, show with a warning message...).
+data:
+  policy_egress: "false"
```

Comment:
What does this data correspond to? Is this just free-form or backed by an enumeration for network features?

Reply:
The names have been chosen freely, though they correspond to real network features. In the current entries:
* `policy_egress`: whether the CNI type supports egress policies.
* `policy_peer_ipblock_exceptions`: whether the CNI type supports setting exceptions to a policy based on peer IP block.

The network features are for internal use (currently, the [Console network policy forms](https://github.com/openshift/console/blob/a28af190fe9bca902b4398b50ee38e89fbdc0f2f/frontend/public/module/k8s/network.ts#L34-L46), but any other naming that fits better your team conventions is accepted.